label/label_selinux.go

```diff
@@ -104,7 +104,13 @@ func Relabel(path string, fileLabel string, relabel string) error {
 	if fileLabel == "" {
 		return nil
 	}
-	if relabel == "z" {
+	if !strings.ContainsAny(relabel, "z & Z") {
```

Comment:
You can do this here otherwise getting a space or & will also satisfy the condition.
```go
if !strings.ContainsAny(relabel, "zZ") {
```